sounds/models.py

```diff
@@ -417,7 +417,9 @@ def duration_ms(self):
         return self.duration * 1000
 
     def rating_percent(self):
-        return int(self.avg_rating*10)
+        if self.num_ratings > settings.MIN_NUMBER_RATINGS:
```

Comment:
Very small style comment. IMHO early return should also be done on the easier path.

```
If self.num_ratings <= ...:
     return 0

return ....
```